builtin_op_importers.cpp

```diff
@@ -882,7 +882,21 @@ DEFINE_BUILTIN_OP_IMPORTER(Gather) {
     nvinfer1::ITensor& data = convertToTensor(inputs.at(0), ctx);
     nvinfer1::ITensor& indices = convertToTensor(inputs.at(1), ctx);
     OnnxAttrs attrs(node);
-    int axis = attrs.get<int>("axis");
+    int axis = attrs.get<int>("axis", 0);
+    // Support negative indexing
+    int ndim = inputs.at(0).shape().nbDims;
+    if (axis < 0)
+    {
+      axis += ndim;
+    }
+
+    // TRT does not support operations across the batch dimension
+    if (inputs.at(0).is_tensor())
+    {
+      --axis; 
+    }
+    
+    ASSERT(0 <= axis && axis < ndim, ErrorCode::kINVALID_NODE);
```

Comment:
This can be simplified by the convert_axis utility function in https://github.com/onnx/onnx-tensorrt/pull/103.